content/_index.en.md

```diff
@@ -284,7 +284,7 @@ The server `<port>` and `<basePath>` are defined in the [configuration](#configu
 
 > For example, with the default configuration, the list of services can be found at [`http://localhost:3000/api/v1/services`](http://localhost:3000/api/v1/services).
 
-## API
+## API [Beta]
```

Comment:
I had in mind that only the Web API was in beta 🤔

Reply:
At this point, I think we can say that both APIs are in beta.